pages.pt_BR/common/caddy.md

```diff
@@ -0,0 +1,28 @@
+# caddy
+
+> Um poderoso servidor web open source, pronto para empresas, com HTTPS automático, escrito em Go.
```

Comment:
Oh, why we have a subjective description here?
```suggestion
> Um servidor web open source, pronto para empresas, com HTTPS automático, escrito em Go.
```
Could you remove `powerful` from the English page too? in this PR